fastats/maths/sum_sq_dev.py

```diff
@@ -15,7 +14,13 @@ def sum_sq_dev(x):
     >>> sum_sq_dev(x) # doctest: +ELLIPSIS
     25681.60000...
     """
-    return sum(power(abs(x - mean(x)), 2))
+    mean = x.mean()
```

Comment:
The idea is that the ast transform in fastats will optionally change standalone functions to be jit versions, so we should aim for `mean(x)` over `x.mean()`, as that then allows us to choose a faster/better version later.